src/test/components/DragAndDrop.test.js

```diff
@@ -124,5 +127,7 @@ describe('app/DragAndDrop', () => {
 
     fireEvent.dragLeave(dragAndDrop);
     expect(overlay.classList).not.toContain('dragging');
+
+    restoreOriginals();
```

Comment:
We don't need to add all these restoreOriginals calls, because `jest.spyOn` spys are automatically removed after each test. See https://github.com/firefox-devtools/profiler/blob/bead77f42b6372bd36579d0d03d9003916b5b504/src/test/setup.js#L25-L28
(historical note: in the past we used to do this with some jest configuration, but this wasn't working properly, see https://github.com/facebook/jest/issues/9896 especially. That's why we're doing it manually in the setup file. I don't know if that improved.)

Reply:
Oh, perfect! That should make the patch a lot simpler.